tutorindigo/templates/indigo/lms/static/js/dark-theme.js

```diff
@@ -1,8 +1,34 @@
 $(document).ready(function() {
     'use strict';
 
-    const enableDarkTheme = {% if INDIGO_ENABLE_DARK_THEME %}true{% else %}false{% endif %};
-    if (enableDarkTheme){
-        $('body').addClass("indigo-dark-theme");
+    {% if INDIGO_THEME_COOKIE_NAME %}
```

Comment:
Remove this condition

Reply:
Based on my understanding, we will proceed by utilizing the ENABLE_DARK_TOGGLE feature. The following outcomes are expected:

- The cookie name is hard-coded throughout the codebase.
- The JavaScript file is compiled during the image-building process.
- The HTML includes the toggle button for enabling or disabling the dark mode.


When the dark-toggle button is enabled or disabled, the following changes will occur:

- The SCSS file will update, setting display: none as required.
- We will remove the inclusion of the dark theme file from the [settings](https://github.com/overhangio/tutor-indigo/pull/97/files#diff-1589d9c9dc5877e40eb63b58cb8de629b62e2ff4c7a7f4782ac2a0cb83806083R190-R194).
- The Micro-Frontend (MFE) will receive the dark theme enablement switch using `MFE_CONFIG['ENABLE_DARK_TOGGLE'] = {True | False}`.

Reply:
Hmmm not quite:

> We will remove the inclusion of the dark theme file from the [settings](https://github.com/edly-io/tutor-indigo/blob/hina/theme-cookies/tutorindigo/plugin.py#L190-L194).

Why is that? Based on your pull request, I don't understand why we would need to do that. You're talking about dark-theme.js, right?

> The Micro-Frontend (MFE) will receive the dark theme enablement switch using MFE_CONFIG['ENABLE_DARK_TOGGLE'] = {True | False}.

Again, I don't think this is necessary -- unless I'm mistaken? We can just hide the toggle using simple css.

Reply:
> Why is that? Based on your pull request, I don't understand why we would need to do that. You're talking about dark-theme.js, right?

When the user disables the switch by setting ENABLE_DARK_TOGGLE=False, the system should start only in the light theme. However, the inclusion of the dark-theme file will still read the theme cookie and add the indigo-dark-theme class to the html body. While the toggle will be hidden, if the user was previously using the dark theme, the file will still retrieve the theme cookie and apply the dark class accordingly.

> Again, I don't think this is necessary -- unless I'm mistaken? We can just hide the toggle using simple CSS.

The use case is as follows: I am a user who has been using the system in toggle mode, and now I disable it. The question is, how will the MFEs know that they need to hide the toggle? We're styling through the brand package. Even if we disable the toggle using ENABLE_DARK_TOGGLE=False, the cookie value still exists. Without a condition in the MFE to prevent loading the theme from the cookie, the system will continue reading and applying the theme based on the cookie value.

Reply:
> When the user disables the switch by setting ENABLE_DARK_TOGGLE=False, the system should start only in the light theme.

I understand. What you are saying is _not_ that we should get rid of dark-theme.js when the toggle is disabled, but instead that we should keep an `{% if ... %}` condition in dark-theme.js. I'm fine with that. We could add that condition to the `applyThemeOnpage` function.